apps/api-extractor/src/analyzer/AstSymbolTable.ts

```diff
@@ -145,43 +146,13 @@ export class AstSymbolTable {
    * or members.  (We do always construct its parents however, since AstDefinition.parent
    * is immutable, and needed e.g. to calculate release tag inheritance.)
    */
-  public analyze(astSymbol: AstSymbol): void {
-    if (astSymbol.analyzed) {
-      return;
+  public analyze(astEntity: AstEntity): void {
```

Comment:
You've opened `AstSymbolTable.analyze` to accept `AstEntity` which includes `AstImport`, but there are no cases that handle it in this function.